modules/home/assets/js/components/icons/youtube-icon.js

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+const YoutubeIcon = () => {
+	return (
+		<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should add it as clean as possible. See e.g. modules/display-conditions/assets/js/editor/components/icons/elementor-logo.js (pro repository)

```suggestion
		<svg viewBox="0 0 24 24"  { ...props }>
```

Reply:
done